### ISSUE TITLE
be able to adjust the transparency level of widgets

### ISSUE THREAD

**Issue by huitre39** [osmandapp/OsmAnd#16017]
Hi osmand community, 
i think it would be nice to be able to adjust the transparency level of the widgets
![Android-osmand-routing](https://user-images.githubusercontent.com/22101022/208660785-3ab0dab9-90f7-47d9-a9fb-1d325e884134.png)

currently in 100% transparent I find it not very readable
if we could set it to 50% for example we could read the text of the widgets better

if we could adjust the level of transparency with a slider for example it would be better, what do you think?

**Comment by scaidermern** [comment]
> currently in 100% transparent I find it not very readable
> if we could set it to 50% for example we could read the text of the widgets better, like the button "+" and "-"

Check the following switch: Configure screen / Transparent widgets.

The result is not the same as the zoom buttons, though. This switch will make the widget background fully transparent while the zoom buttons use something in-between (maybe 50%, I don't know).

**Comment by arncor** [comment]
Almost the same as #14957

**Comment by snooppr** [comment]
Around the topic. On sexy screens (small screen sizes), widgets and buttons take up a lot of space, and maps don't.  It would be great if the developers could add a feature to "**temporarily hide widgets/buttons**" leaving only the map, such functionality is available to competitors in organicmaps.

How to implement.  Add to settings: toggle "use volume keys to hide buttons?"  volume down key - hide buttons and widgets, volume up key - show buttons and widgets.

**Comment by scaidermern** [comment]
> It would be great if the developers could add a feature to "**temporarily hide widgets/buttons**" leaving only the map

This is exactly what full screen mode does. Enable settings / profile / general settings / fullscreen mode. Then simply tap once onto the map to enable it. This will hide all UI elements. Tap again to disable it.

**Comment by snooppr** [comment]
> 

In the latest version of osmand f-droid this function does not work.  After activating it and even restarting the application, nothing happens on clicks.  Perhaps activated other functions are blocking it?

**Comment by scaidermern** [comment]
> It works, but only for one "map overview" profile, it does not work for other profiles: on foot, boat, bicycle. Android 9.

This setting is profile specific. When you change it OsmAnd should ask you whether to change it for the current profile only or for all profiles.

**Comment by snooppr** [comment]
> > It works, but only for one "map overview" profile, it does not work for other profiles: on foot, boat, bicycle. Android 9.
> 
> This setting is profile specific. When you change it OsmAnd should ask you whether to change it for the current profile only or for all profiles.

Of course, I know, I applied for one profile and for all.  And once again I will write the bug is present, it does not work, or rather it works only for one user.

**Comment by scaidermern** [comment]
> Of course, I know, I applied for one profile and for all. And once again I will write the bug is present, it does not work, or rather it works only for one profile.

Do you have Configure screen / distance by tap enabled? If this is enabled then full screen mode doesn't work. Not sure if there are other feature interactions.

**Comment by snooppr** [comment]
> > Of course, I know, I applied for one profile and for all. And once again I will write the bug is present, it does not work, or rather it works only for one profile.
> 
> Do you have Configure screen / distance by tap enabled? If this is enabled then full screen mode doesn't work. Not sure if there are other feature interactions.


Indeed, I have the option enabled:"distance by tap enabled", which is located in another block of options and it blocked the functionality, I need it, and therefore is incompatible with the option: "hide widgets".
I still believe that you can bind an option to the volume keys without conflicting with other parameters.

Thanks anyway, this worked out.

**Comment by dmpr0** [comment]
Duplicate https://github.com/osmandapp/OsmAnd/issues/14957
